lib/std/zig/Ast.zig

```diff
@@ -458,6 +458,12 @@ pub fn renderError(tree: Ast, parse_error: Error, stream: anytype) !void {
             return stream.writeAll("for input is not captured");
         },
 
+        .invalid_byte => {
+            return stream.print("string/comment contains invalid byte: '{'}'", .{
+                std.zig.fmtEscapes(tree.source[(tree.tokens.items(.start)[parse_error.token] + parse_error.extra.offset)..][0..1]),
+            });
```

Comment:
We know which it is, sorry if I was not clear earlier.
```suggestion
            const tok_slice = tree.source[tree.tokens.items(.start)[parse_error.token]..];
            return stream.print("{s} contains invalid byte: '{'}'", .{
                switch (tok_slice[0]) {
                    '\'' => "character literal",
                    '"', '\\' => "string literal",
                    '/' => "comment",
                    else => unreachable,
                },
                std.zig.fmtEscapes(tok_slice[parse_error.extra.offset..][0..1]),
            });
```

Reply:
That makes way more sense, sorry for misreading earlier!